linker/spm.eu0.lst

```diff
@@ -6,7 +6,8 @@
 // more
 
 /* Dol */
-
+8045B9E0:superDimentioHeadDefenses
```

Comment:
These should be moved under npc_dimeen_l.c

Reply:
all done!